tests/suites/test_suite_psa_crypto_slot_management.function

```diff
@@ -487,7 +484,20 @@ void copy_across_lifetimes( int source_lifetime_arg, int source_id_arg,
     /* Test that the target slot has the expected content. */
     psa_reset_key_attributes( &target_attributes );
     PSA_ASSERT( psa_get_key_attributes( target_handle, &target_attributes ) );
-    TEST_EQUAL( target_id, psa_get_key_id( &target_attributes ) );
+
+    if( target_lifetime != PSA_KEY_LIFETIME_VOLATILE )
+    {
+        TEST_ASSERT( PSA_KEY_ID_EQUAL(
+                         target_id, psa_get_key_id( &target_attributes ) ) );
+    }
+    else
+    {
+        #if defined(MBEDTLS_PSA_CRYPTO_KEY_EXTENDED_ID)
+        TEST_EQUAL( PSA_KEY_ID_GET_OWNER_ID( target_id ), 1 );
+        #endif
+        TEST_EQUAL( PSA_KEY_ID_GET_ID( target_id ) , 0 );
```

Comment:
Because we need to construct the (1,0) identifier and then compare it with `target_id`. I can do it if you prefer, not done yet.

Reply:
I was wondering because in other places, you do use `PSA_KEY_ID_EQUAL`, and I couldn't find a reason for doing it differently.

I see advantages to both approaches. `PSA_KEY_ID_EQUAL` respects the abstraction, so it's more robust. Testing the two parts separately will give a better error message once we finally make `TEST_EQUAL` print the values.